Xamarin.Forms.Core.UnitTests/MultiBindingTests.cs

```diff
@@ -235,22 +223,22 @@ public void TestConverterReturnValues()
 
 			var label2 = GenerateNameLabel(nameof(group.Person2), BindingMode.TwoWay);
 			stack.Children.Add(label2);
-			label2.Text = $"DoNothing {oldMiddleName} {oldLastName.ToUpper()}";
+			label2.SetValueCore(Label.TextProperty, $"DoNothing {oldMiddleName} {oldLastName.ToUpper()}", Internals.SetValueFlags.None);
 			Assert.AreEqual($"{oldFirstName} {oldMiddleName} {oldLastName.ToUpper()}", group.Person2.FullName);
-			Assert.AreEqual(group.Person2.FullName, label2.Text);
+			Assert.AreEqual($"DoNothing {oldMiddleName} {oldLastName.ToUpper()}", label2.Text);
```

Comment:
in case when some conversion use DoNothing or UnsetValue, do not expect your explicitly set Label.Text to be changed. that'd be weird.

changed (here and at some other places) tests to make sure the set value is the one on display, even if the converter prevents binding propagation